### PR TITLE
feat: added decoding functions calls

### PR DESCRIPTION
**Pull request by RahulSankhla312**
This pull request addresses the issue titled "Show how many times a specific decoding function has been called" (https://github.com/mandiant/flare-floss/issues/693).

Please review the changes I've implemented. I welcome any suggestions for necessary adjustments.

### CODE REVIEW

**Comment by RahulSankhla312** [comment]
@mr-tz Please review this pr , i have only kept the minimal changes.

**Comment by mr-tz** [comment]
can you please fix the black fails?

**Comment by RahulSankhla312** [comment]
@mr-tz I have fixed the black fails .

**Comment by mr-tz** [comment]
mypy outstanding:

```
floss/main.py:769: error: Dict entry 0 has incompatible type "str": "Any"; expected "int": "float"  [dict-item]
floss/main.py:769: error: Dict entry 1 has incompatible type "str": "Any"; expected "int": "float"  [dict-item]
Found 2 errors in 1 file (checked 56 source files)
```

**Comment by mr-tz** [comment]
great, now only need to update the tests at https://github.com/mandiant/flare-floss/blob/73020a427bdcc65062a2fafecf89ff1f1cc3d865/tests/test_load.py#L5

**Comment by RahulSankhla312** [comment]
@mr-tz i have added some test cases for test_load.py

File: floss/main.py [hunk 1]
```diff
@@ -783,8 +783,10 @@ def main(argv=None) -> int:
             else:
                 logger.debug("identified %d candidate decoding functions", len(fvas_to_emulate))
                 for fva in fvas_to_emulate:
-                    results.analysis.functions.decoding_function_scores[fva] = decoding_function_features[fva]["score"]
-                    logger.debug("  - 0x%x: %.3f", fva, decoding_function_features[fva]["score"])
+                    score = decoding_function_features[fva]["score"]
+                    xrefs_to = decoding_function_features[fva]["xrefs_to"]
+                    results.analysis.functions.decoding_function_scores[fva] = {"score": score, "xrefs_to": xrefs_to}
+                    logger.debug("  - 0x%x: score: %.3f, calls: %d", fva, score, xrefs_to)
```
Comment (mr-tz) [inline floss/main.py:789]:
```suggestion
                    logger.debug("  - 0x%x: score: %.3f, xrefs to: %d", fva, score, xrefs_to)
```

**Review by mr-tz — APPROVED** [review]
thank you very much! nice work!

**Pull request merged**